intelligence apps: add project update and update profiling (#9879)
Microsoft/vscode-docs: docs/intelligentapps/profiling.md

![Screenshot that shows the report data for each OP](./images/profiling/by-model-file-op-result.png)

#### Enable op level data for Intel (OpenVINO) EP

To see op level data when running on the Intel (OpenVINO) EP, do the following:

> - Download [Intel® Unified Telemetry](https://www.intel.com/content/www/us/en/developer/tools/intel-unified-telemetry.html)

Reviewer: Make this a numbered list if the order is important

Author: changed

- Extract the content to a path like `C:\Users\XXX\Downloads\ut-tool-ext-v0.2.0-beta1.1`
- In Foundry Toolkit settings, set `Model Lab Intel Unified Telemetry Path` to `C:\Users\XXX\Downloads\ut-tool-ext-v0.2.0-beta1.1\bin`

> Intel Unified Telemetry requires administrator privileges to run. If VS Code is not running as administrator, profiling will prompt for elevation and open a new terminal window showing its progress; please wait and do not close that window.

Reviewer: ```suggestion
Intel Unified Telemetry requires administrator privileges to run. If VS Code is not running as administrator, profiling will prompt for elevation and open a new terminal window showing its progress. Don't close this window until the process has completed.
```
